### ISSUE TITLE
About the evaluation script on the validation set

### ISSUE THREAD

**Issue by davidyang180** [timmeinhardt/trackformer#57]
Hello! I saw in the paper that the evaluation index of MOTA is 71.3 reported on the validation set in the ablation experiment. I found that there are only evaluation scripts for the test set and training set in the code. May I ask when I want to evaluate on the validation set, I need to manually select Related video sequences?

**Comment by timmeinhardt** [comment]
The cross-validation sets are a per sequence frame split. To only evaluate a subset of frames the `src/track.py` has a `frame_range` parameter. This means you can run `python src/track.py with frame_range.start=0.5`.

**Comment by davidyang180** [comment]
Thank you for your reply. For the ablation experiments in some MOT papers, half of the training and half are used as the validation set. In your paper, do you use all of them for training, and then use the latter half for the validation of the ablation experiments?

**Comment by davidyang180** [comment]
> There is no dedicated validation set. We validate on a cross-validation split on the training sequences. The split is a per -sequence frame split. See for example `mot17_train_cross_val_frame_0_0_to_0_5_coco` in `src/generate_coco_from_mot.py`. To only evaluate a subset of frames the `src/track.py` has a `frame_range` parameter. This means you can run `python src/track.py with frame_range.start=0.5`.

Thank you for your reply. For the ablation experiments in some MOT papers, half of the training and half are used as the validation set. In your paper, do you use all of them for training, and then use the latter half for the validation of the ablation experiments?

**Comment by timmeinhardt** [comment]
For the ablation experiments, we use do 50-50 frame split on the training sequences. Training on the first 50% of the frames of each training sequences and validating on the second 50%. For our final submission and test set evaluation we train on all frames of the training sequences.

The [arxiv](https://arxiv.org/pdf/2101.02702.pdf) paper has the appendix at the end.
